test/integration_test.js

```diff
@@ -178,51 +194,223 @@ describe("Integration tests", () => {
+          it("should put local database in the expected state", () => {
+            return tasks.list({order: "title"})
+              .then(res => res.data.map(record => ({
+                title: record.title,
+                done: record.done,
+                _status: record._status,
+              })))
+              .should.become([
+                {title: "task1", _status: "synced", done: true},
+                {title: "task2", _status: "synced", done: false},
+                {title: "task3", _status: "synced", done: true},
+                // For MANUAL strategy, local conficting record is left intact
+                {title: "task4-local", _status: "created", done: false},
+              ]);
+          });
         });
 
-        it("should not have published anything", () => {
-          expect(syncResult.published).to.have.length.of(0);
+        describe("CLIENT_WINS strategy", () => {
+          beforeEach(() => {
+            return testSync(testData, {
+              strategy: Kinto.syncStrategy.CLIENT_WINS
+            }).then(res => syncResult = res);
+          });
+
+          it("should not have an ok status", () => {
+            expect(syncResult.ok).eql(true);
+          });
+
+          it("should contain no errors", () => {
+            expect(syncResult.errors).to.have.length.of(0);
+          });
+
+          it("should have a valid lastModified value", () => {
+            expect(syncResult.lastModified).to.be.a("number");
+          });
+
+          it("should have no incoming conflict listed", () => {
+            expect(syncResult.conflicts).to.have.length.of(0);
+          });
+
+          it("should not skip records", () => {
+            expect(syncResult.skipped).to.have.length.of(0);
+          });
+
+          it("should not import anything", () => {
+            expect(syncResult.created).to.have.length.of(0);
+          });
+
+          it("should publish resolved conflict using local version", () => {
+            expect(syncResult.published).to.have.length.of(1);
+            expect(cleanRecord(syncResult.published[0])).eql({
+              id: conflictingId,
+              title: "task4-local",
+              done: false,
+            });
+          });
+
+          it("should update conflict with local version", () => {
+            expect(syncResult.updated).to.have.length.of(1);
+            expect(cleanRecord(syncResult.updated[0])).eql({
+              id: conflictingId,
+              title: "task4-local",
+              done: false,
+            });
+          });
+
+          it("should list resolved records", () => {
+            expect(syncResult.resolved).to.have.length.of(1);
+            expect(cleanRecord(syncResult.updated[0])).eql({
+              id: conflictingId,
+              title: "task4-local",
+              done: false,
+            });
+          });
+
+          it("should put local database in the expected state", () => {
```

Comment:
I would probably add a test to make sure that the conflict resolution was send to the remove server.